articles/130_ros_time.md

```diff
@@ -140,6 +144,15 @@ The underlying datatypes will also provide ways to register notifications, howev
   <b>TODO:</b> Enumerate the <code>rcl</code> datastructures and methods here.
 </div>
 
+## Work Needing To Be Done (Apr 2017)
+
+- Decide where and when to autosubscribe to the `/clock` message. Built in to Node? A Node addon or service? Part of the first call to `rcl_node_init`?
```

Comment:
I think this builds on the comment you made elsewhere (can't find it right now), that Time needs to be somehow connected to a Node so it can subscribe to this topic. I think you're right about that, and it's something we have to consider now because in ROS 1 there was an omnipresent node which could be used for this purpose (the node singleton). We'll have to find a way to connect ROS Time to a node when sim time is desired.

I'm not sure how that should work atm. But it is definitely related to the mapping between ROS nodes and DDS participants and also how we manage multiple nodes in a single process, i.e. how the "container node" which loads and runs all the other nodes is implemented.